src/Microsoft.Extensions.Logging.AzureWebApps/Internal/IWebAppsContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Extensions.Logging.AzureWebApps.Internal
+{
+    /// <summary>
+    /// Represents an Azure WebApp context
+    /// </summary>
+    public interface IWebAppsContext
+    {
+        /// <summary>
+        /// Gets the path to the home folder if running in Azure WebApps
+        /// </summary>
+        string HomeFolder { get; }
+
+        /// <summary>
+        /// Gets a value indicating whether or new we're in an Azure WebApp
+        /// </summary>
+        bool IsRunningInAzureWebApp { get; }
```

Comment:
Why do we want such a restriction? (example if a user wants to use this in their application hosting provider outside azure, they are not supposed to use it?) is it to discourage users from using this logger for non-web apps scenario?